### PR TITLE
Add support for python3, remove support for python2

### PR DESCRIPTION
**Pull request by pazembrz**

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -40,6 +39,18 @@ def test_get_image_location_ok(tmpdir):
...
 
 
+def test_get_image_location_unicode_path_ok(tmpdir):
+    image = "ąþð/→µŋ.png"
+    path = six.text_type(tmpdir.mkdir('ąþð').join("→µŋ.png"))
+    image_list = [path]
+
+    assert path == plotextractor.output_utils.get_image_location(
+        image,
+        six.text_type(tmpdir),
```
Comment (drjova) [inline tests/test_utils.py]: why we need to do ``six`` here?

Comment (pazembrz) [reply tests/test_utils.py]: We don't need it anymore at all, but it's also not harmful at all. Please notice that it's in many places and I didn't want to loose time on removing it. Of course I can do it if you wish.

File: tests/test_utils.py [hunk 2]
```diff
@@ -40,6 +39,18 @@ def test_get_image_location_ok(tmpdir):
     )
 
 
+def test_get_image_location_unicode_path_ok(tmpdir):
```
Comment (michamos) [inline tests/test_utils.py]: can you also add a test where you put a file path that is **not** valid UTF-8?

File: plotextractor/converter.py [hunk 3]
```diff
@@ -24,15 +24,14 @@
...
-from __future__ import absolute_import, print_function
 
 import os
 import tarfile
 import re
 
 from time import time
 
-from subprocess32 import check_output, TimeoutExpired
+from subprocess32 import check_output
```
Comment (michamos) [inline plotextractor/converter.py]:
```suggestion
from subprocess import check_output
```
subproess32 is a backport of python 3 subprocess to python 2.

File: tests/test_utils.py (hunk 2 above)
Comment (pazembrz) [reply tests/test_utils.py]: Done :+1:

**Pull request merged**